File: Python/Toko.py
from Perusahaan import Perusahaan

class Toko(Perusahaan):
    def __init__(self, pemilik, nib, lokasi, namaToko, penghasilan, jamOperasional):
        super().__init__(pemilik, nib, lokasi)
        self._namaToko = namaToko
        self._penghasilan = penghasilan
        self._jamOperasional = jamOperasional

    # Setter
    def setNamaToko(self, v): self._namaToko = v
    def setPenghasilan(self, v): self._penghasilan = v
    def setJamOperasional(self, v): self._jamOperasional = v

    # Getter
    def getNamaToko(self): return self._namaToko
    def getPenghasilan(self): return self._penghasilan
    def getJamOperasional(self): return self._jamOperasional

File: Java/Toko.java
public class Toko extends Perusahaan {
    private String namaToko;
    private double penghasilan;
    private String jamOperasional;

    public Toko(String pemilik, String nib, String lokasi,
                String namaToko, double penghasilan, String jamOperasional) {
        super(pemilik, nib, lokasi);
        this.namaToko = namaToko;
        this.penghasilan = penghasilan;
        this.jamOperasional = jamOperasional;
    }

    // Setter
    public void setNamaToko(String v) { namaToko = v; }
    public void setPenghasilan(double v) { penghasilan = v; }
    public void setJamOperasional(String v) { jamOperasional = v; }

    // Getter
    public String getNamaToko() { return namaToko; }
    public double getPenghasilan() { return penghasilan; }
    public String getJamOperasional() { return jamOperasional; }
}
